.travis.yml

```diff
@@ -4,12 +4,12 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27-sdist
-    - python: 3.6
-      env: TOXENV=py36-sdist
     - python: 3.7
       env: TOXENV=py37-sdist
     - python: 3.8
-      env: TOXENV=py38-sdist,formatting,docs
+      env: TOXENV=py38-sdist
+    - python: 3.11
+      env: TOXENV=py311-sdist,formatting,docs
```

Comment:
It doesn't look like Travis has actually run against this pull request? Perhaps because we haven't made a change in a while. We could switch this to github actions?